### PR TITLE
docs: Document setting metadata about a package's authors

### PR DESCRIPTION
**Pull request by bennati**
Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: website/docs/configuration/package-curations.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ The structure of the curations file consist of one or more `id` entries:
   curations:
     comment: "An explanation why the curation is needed or the reasoning for a license conclusion"
     purl: "pkg:Maven/com.example.app/example@0.0.1?arch=arm64-v8a#src/main"
+    authors: ["The names of the authors"]
```
Comment (sschuberth) [inline website/docs/configuration/package-curations.md:60]: In YAML, a list needs to be declared as follows:

```
  authors:
  - "Name of one author"
  - "Name of another author"
```

File: website/docs/configuration/package-curations.md [hunk 2]
```diff
@@ -65,12 +66,12 @@ The structure of the curations file consist of one or more `id` entries:
     homepage_url: "http://example.com"
     binary_artifact:
       url: "http://example.com/binary.zip"
-      hash: 
+      hash:
```
Comment (sschuberth) [inline website/docs/configuration/package-curations.md:69]: Ideally, do the trimming in a separate commit, as it's a non-functional change.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: website/docs/configuration/package-curations.md (hunk 1 above)
Comment (bennati) [reply website/docs/configuration/package-curations.md]: done

File: website/docs/configuration/package-curations.md (hunk 2 above)
Comment (bennati) [reply website/docs/configuration/package-curations.md]: reverted

Comment (sschuberth) [reply website/docs/configuration/package-curations.md]: No need to revert, the change is good per se, but simply should go into a separate / preceding commit.

File: website/docs/configuration/package-curations.md (hunk 1 above)
Comment (bennati) [reply website/docs/configuration/package-curations.md]: done

File: website/docs/configuration/package-curations.md (hunk 2 above)
Comment (bennati) [reply website/docs/configuration/package-curations.md:71]: done

**Review by sschuberth — APPROVED** [review]
